File: app/src/main/java/org/stepic/droid/ui/decorators/SpacesItemDecorationHorizontal.java
package org.stepic.droid.ui.decorators;

import android.graphics.Rect;
import android.support.v7.widget.RecyclerView;
import android.view.View;

public class SpacesItemDecorationHorizontal extends RecyclerView.ItemDecoration {
    private int space;

    public SpacesItemDecorationHorizontal(int space) {
        this.space = space;
    }

    @Override
    public void getItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state) {
        outRect.right = space;
        if (parent.getChildPosition(view) == 0)
            outRect.left = space;
    }
}

File: app/src/main/java/org/stepic/droid/ui/decorators/SpacesItemDecorationHorizontal.kt
package org.stepic.droid.ui.decorators

import android.graphics.Rect
import androidx.recyclerview.widget.RecyclerView
import android.view.View

class SpacesItemDecorationHorizontal(private val space: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        outRect.right = space
        if (parent.getChildAdapterPosition(view) == 0) {
            outRect.left = space
        }
    }
}
